README.md

```diff
@@ -42,6 +43,41 @@ Below is a boilerplate HTML file to get you started:
         font-weight: normal;
       }
       .remark-code, .remark-inline-code { font-family: 'Ubuntu Mono'; }
+      
+      /*
+       * Transitions between slides by using Animate.css
+       * Find interesting transition animations here!
+       * https://daneden.github.io/animate.css/
+       */  
+       
+       /* Browsing forwards transition */
+    
+       /* Browse to slide */
+       .remark-visible.remark-browsing-backwards {
+           animation: slideInDown 0.5s forwards;
+       }
+       /* Browse from slide */
+         .remark-fading.remark-browsing-backwards {
```

Comment:
The indentation is a bit off on some lines. It's a fantastic change otherwise! I hope it gets merged.